### ISSUE TITLE
Wolf prompts for 'hunter' install

### ISSUE THREAD

**Issue by 0916174** [Duroktar/Wolf#1]
Hello there,

I was very interested in Wolf, since it seems really cool and want to see what I can do.
I'm having a problem though.

After installing it via the VS Extension Manager, I restarted VSCode and, after pressing ⌘(Cmd)+Shift+P (same as F1 on Mac) and selecting "Wolf: Set the Wolf on the current file.", it prompted me to install hunter. So I do so in my terminal window. Then I restarted, but Wolf still gives me the same prompt after activating Wolf.

<img width="831" alt="schermafbeelding 2018-02-07 om 23 56 30" src="https://user-images.githubusercontent.com/14292245/35946300-85b6d0da-0c63-11e8-8969-861140996aeb.png">
<img width="742" alt="schermafbeelding 2018-02-07 om 23 57 30" src="https://user-images.githubusercontent.com/14292245/35946301-85cf52f4-0c63-11e8-9da6-351f68b8f629.png">
<img width="673" alt="schermafbeelding 2018-02-08 om 00 04 13" src="https://user-images.githubusercontent.com/14292245/35946335-a8535bf4-0c63-11e8-90e3-82b2689b0119.png">

Do note, I'm using a different interpreter than the default one that VS Code selects for me. I've set my interpreter using ⌘(Cmd)+Shift+P and choosing "Python: Select Interpreter".
It's currently set to `/usr/local/bin/python3`.

<img width="607" alt="schermafbeelding 2018-02-08 om 00 06 32" src="https://user-images.githubusercontent.com/14292245/35946432-01f0525c-0c64-11e8-9397-d78160b89c9d.png">

_Edit: I can confirm the `python3` in my terminal is the same as the selected interpreter in VS Code, as shown below._
<img width="517" alt="schermafbeelding 2018-02-08 om 00 08 10" src="https://user-images.githubusercontent.com/14292245/35946495-44c077d8-0c64-11e8-9128-981f57e686ff.png">

Thank you for your work and help.

Best regards,
Jeroen.

**Comment by Duroktar** [comment]
Hmm, that is odd. When I first started adding the notification for Hunter the same thing happened to me; I put in a button on that notification that will install Hunter for you (it actually installs to the `--user` directory, so no need for sudo). In my case, at least on my main (Arch linux) pc and (Mac) laptop, that was sufficient to make it work.

I'm going to try to reproduce your issue on my end with a fresh Python install. In the meanwhile, would you be willing to check if clicking the "Install Hunter" button fixes it? I'll let you know how it goes. Thanks again for the interest in my project, and sorry about the troubles, but these growing pains are expected and I appreciate the feedback _very_ much.

Cheers

**Comment by Duroktar** [comment]
Hey there! Are you using `pyenv` by chance? I _think_ that I was able to reproduce the problem but can't be sure it was the same thing. On the other hand, I was looking into the python API that is now built in and I think that I can use that to ensure the correct Python binary is being run. I'm busy the first part of today, but I'll try to have an update by tonight using that.

Thanks again 🥂

**Comment by 0916174** [comment]
Hi there,

I’m not home right now. I’ll message you when I have an answer.

No problem. I know how difficult debugging deployment can be ;)

I don’t know what you mean by pyenv. Then again, I’m a tad tipsy, so I’ll reply whenever I have new information.

Best regards

Op 8 feb. 2018 om 15:06 heeft traBpUkciP <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Hey there! Are you using pyenv by chance? I think that I was able to reproduce the problem but can't be sure it was the same thing. On the other hand, I was looking into the python API that is now built in and I think that I can use that to ensure the correct Python binary is being run. I'm busy the first part of today, but I'll try to have an update by tonight using that.

Thanks again 🥂

**Comment by Duroktar** [comment]
Oops, I don't remember closing this. I think GitHub did for me cause I mentioned #1 in the last commit message. 

Anyways. I tweaked the code around last night, I was injecting the script path into the Python shell and think it might have been overriding some local settings, so it's now removed. It seems to allow me to go through the install procedure that you went through and not complain about finding Hunter so if you can give it a shot at some point I'd be very interested to know if it fixes it for you too.

I hope ya had a good one :) I ended up having a couple pops last night also, so I hope I didn't mess anything up too bad, haha.

Cheers

**Comment by 0916174** [comment]
Hi,

I was already reading the commit comment through and didn’t see anything related to my issue. Glad to know I was right.

I’ll try it out tomorrow or sooner if I can find the time.

Nice, cheers indeed ;)

Best regards.

Op 9 feb. 2018 om 16:57 heeft traBpUkciP <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


Oops, I don't remember closing this. I think GitHub did for me cause I mentioned #1<https://github.com/Duroktar/Wolf/issues/1> in the last commit message.

Anyways. I tweaked the code around last night, I was injecting the script path into the Python shell and think it might have been overriding some local settings, so it's now removed. It seems to allow me to go through the install procedure that you went through and not complain about finding Hunter so if you can give it a shot at some point I'd be very interested to know if it fixes it for you too.

I hope ya had a good one :) I ended up having a couple pops last night also, so I hope I didn't mess anything up too bad, haha.

Cheers

**Comment by Duroktar** [comment]
Yeah, no you were definitely right. Github is a little to smart when I put a commit message like:
```
  removed cwd argument from python spawner, hopefully this fixes #1
```

It saw the "fixes" and "#1" so it thought it was closed. So, anyways, yeah sometime when you get a chance give it a rip. I'll keep the issue open until you're happy :)

:beers:

**Comment by Duroktar** [comment]
Closing for now. Feel free to re-open it anytime though. Thanks again!

**Comment by 0916174** [comment]
Hey there, sorry for not getting back to you for so long, other things got in the way.
Got a new Mac since last time, and I'm getting the same error again. I've installed hunter in all ways I know of, `pip install, pip3 install, python3 -m pip install..`, etc.
I've changed my python environment within VS Code to `python3`:
![image](https://user-images.githubusercontent.com/14292245/39627271-9258c674-4fa5-11e8-8105-41a0e5764f36.png)
